tests/docstring_tests/DocTest.res

```diff
@@ -141,9 +141,9 @@ let main = async () => {
         } else {
           // Let's add the examples inside a Test module because some examples
           // have type definitions that are not supported inside a block.
-          // Also add unit type `()`
+          // Also add unit type `()`. Also wrap in async to make top-level awaits work.
```

Comment:
This might be controversial, but all tests pass.

Reply:
I assume there is no noticeable performance difference from this?

Reply:
Nope:


master
```sh
node scripts/test.js -docstrings  8.49s user 2.25s system 365% cpu 2.938 total
```

fix-result-examples
```sh
node scripts/test.js -docstrings  8.44s user 2.04s system 357% cpu 2.933 total
```